### PR TITLE
refs #11648 - ensure postgresql is started

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: hooks/pre/30-upgrade.rb [hunk 1]
```diff
@@ -1,17 +1,23 @@
+def start_service(service)
+  status = Kafo::Helpers.execute("service-wait #{service} status")
```
Comment (ehelms) [inline hooks/pre/30-upgrade.rb]: Guessing katello-service doesn't have a start --only type of option such that we aren't relying on two service interfaces?

Comment (stbenjam) [inline hooks/pre/30-upgrade.rb]: We could, I'll have to check. Mostly worried about what upstart does when you try to start an already started service, I think that's why I check the status first here.  systemd will return 0, but all of the old init scripts might not play so nicely.

**Comment by stbenjam** [comment]
Updated to use @bbuckingham's `--only`

**Comment by ehelms** [comment]
ACK

**Pull request merged**